src/main/java/com/jcraft/jsch/OpenSSHConfig.java

```diff
@@ -74,6 +78,11 @@
  */
 public class OpenSSHConfig implements ConfigRepository {
 
+  private static final Set<String> keysWithListAdoption = Stream
+      .of("KexAlgorithms", "Ciphers", "HostbasedAcceptedAlgorithms",
+          "HostKeyAlgorithms", "MACs", "PubkeyAcceptedAlgorithms", "CASignatureAlgorithms", "PubkeyAcceptedKeyTypes")
```

Comment:
Should we keep the keywords in here that aren't currently supported (like `HostbasedAcceptedAlgorithms` & `CASignatureAlgorithms`)?

Reply:
correct. I removed them